aws/resource_aws_db_parameter_group_test.go

```diff
@@ -539,6 +548,45 @@ func testAccCheckAWSDBParameterGroupExists(n string, v *rds.DBParameterGroup) re
 	}
 }
 
+func testAccCheckAWSDBParameterNotUserDefined(n, paramName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No DB Parameter Group ID is set")
+		}
+
+		conn := testAccProvider.Meta().(*AWSClient).rdsconn
+
+		opts := rds.DescribeDBParametersInput{
+			DBParameterGroupName: aws.String(rs.Primary.ID),
+			Source:               aws.String("user"),
+		}
+		fmt.Printf("test: %s\n", rs.Primary.ID)
```

Comment:
In general, we'd prefer not to have `fmt.Printf`s in the code. For potential debug information, a `log.Printf` could work, but if these were just testing messages, could you go ahead and remove them?